Snakefile

```diff
@@ -1068,8 +1907,12 @@ rule run_scenario:
         )
         # merge the default config file with the difference
         create_test_config(base_config_path, input.diff_config, "config.yaml")
-        os.system("snakemake -j all solve_all_networks --rerun-incomplete")
-        os.system("snakemake -j1 make_statistics --force")
+        run(
+            "snakemake -j all solve_all_networks --rerun-incomplete",
+            shell=True,
+            check=True,
+        )
+        run("snakemake -j1 make_statistics --force", shell=True, check=True)
```

Comment:
What is the benefit if the run() function here?

Reply:
using the run function streams the error correctly, meaning whenever snakemake fails, the `run` function will also fail. this was not the case before which lead to the weird behavior that the snakemake run failed but the process still finished without an error

Reply:
To be honest that behavior was wanted. This seemed a new feature to me that may not have needed to be merged now.
I'll personally disable the feature for my own needs, but I know how to do that in case ;)
We can decide what is the expected behavior for the general user and agree on that

Reply:
Yes, we talked about it in a one-by-one chat and I forgot about that point. So, thanks for comming back to that! I have added a flag, as discussed, so that the user can trigger the loose and strict behaviour. default is now strict.